cea/optimization/slave/least_cost.py

```diff
@@ -91,11 +91,16 @@ def least_cost_main(locator, master_to_slave_vars, solar_features, gv):
     # Import Temperatures from Network Summary:
 
     os.chdir(locator.get_optimization_network_results_folder())
-    tdhret = np.array(pd.read_csv(NETWORK_DATA_FILE, usecols=["T_sst_heat_return_netw_total"],
-                                  nrows=gv.DAYS_IN_YEAR * gv.HOURS_IN_DAY))
-    mdot_DH = np.array(
-        pd.read_csv(NETWORK_DATA_FILE, usecols=["mdot_DH_netw_total"], nrows=gv.DAYS_IN_YEAR * gv.HOURS_IN_DAY))
-    tdhsup = np.array(pd.read_csv(NETWORK_DATA_FILE, usecols=["T_sst_heat_supply_netw_total"], nrows=1))[0][0]
+    NETWORK_DATA = pd.read_csv(NETWORK_DATA_FILE)
+    tdhret = NETWORK_DATA['T_sst_heat_return_netw_total']
+
+    mdot_DH = NETWORK_DATA['mdot_DH_netw_total']
+    tdhsup = NETWORK_DATA['T_sst_heat_supply_netw_total'][0]
+    # tdhret = np.array(pd.read_csv(NETWORK_DATA_FILE, usecols=["T_sst_heat_return_netw_total"],
```

Comment:
erase commented code if its not expected to be used in the future. It might be confusing fro new users